iothub_client/inc/iothub_device_client_ll.h

```diff
@@ -374,6 +374,108 @@ typedef struct IOTHUB_CLIENT_CORE_LL_HANDLE_DATA_TAG* IOTHUB_DEVICE_CLIENT_LL_HA
 
 #endif /*DONT_USE_UPLOADTOBLOB*/
 
+/**
+* @brief    Asynchronous call to send the telemetry message specified by @p telemetryMessageHandle.
+*
+* @param[in]    iotHubClientHandle              The handle created by a call to the create function.
+* @param[in]    telemetryMessageHandle          The handle to an IoT Hub message.
+* @param[in]    telemetryConfirmationCallback   Optional callback specified by the device for receiving
+*                                               confirmation of the delivery of the telemetry.
+* @param[in]    userContextCallback             Optional user specified context that will be provided to the callback.
+*
+* @remarks
+*           The application behavior is undefined if the user calls
+*           the IoTHubDeviceClient_LL_Destroy function from within any callback.
+*
+*           The IOTHUB_MESSAGE_HANDLE instance provided as argument is copied by the function,
+*           so this argument can be destroyed by the calling application right after IoTHubDeviceClient_LL_SendTelemetryAsync returns.
+*           The copy of @p telemetryMessageHandle is later destroyed by the iothub client when the message is effectively sent, if a failure sending it occurs, or if the client is destroyed.
+*
+* @return   IOTHUB_CLIENT_OK upon success or an error code upon failure.
+*/
+IOTHUB_CLIENT_RESULT IoTHubDeviceClient_LL_SendTelemetryAsync(
+                        IOTHUB_DEVICE_CLIENT_LL_HANDLE iotHubClientHandle,
+                        IOTHUB_MESSAGE_HANDLE telemetryMessageHandle,
+                        IOTHUB_CLIENT_TELEMETRY_CALLBACK telemetryConfirmationCallback,
+                        void* userContextCallback);
+
+/**
+* @brief    Subscribes to incoming commands from IoT Hub.
+*
+* @param[in]  iotHubClientHandle                 The handle created by a call to the create function.
+* @param[in]  commandCallback                    The callback which will be called when a command request arrives.
+* @param[in]  userContextCallback                Optional specified context that will be provided to the callback.
```

Comment:
Nit: Remove unneeded space?
```suggestion
* @param[in]  iotHubClientHandle        The handle created by a call to the create function.
* @param[in]  commandCallback           The callback which will be called when a command request arrives.
* @param[in]  userContextCallback       Optional specified context that will be provided to the callback.
```